tools/bumper/component_commands_test.go

```diff
@@ -325,62 +389,71 @@ var _ = Describe("Testing internal git", func() {
 			currentReleaseVersion: "v2.5.1",
 			latestReleaseVersion:  "v3.6.2",
 			updatePolicy:          "static",
+			prTitle:               "dummy new PR title",
```

Comment:
nit: I think a const can be used here

Reply:
[Done](https://github.com/kubevirt/cluster-network-addons-operator/compare/925a49e92daed1a59942412539cfd5a20cb05601..62c810a5cec404a8115a802800c2aab5c0f87ed7)